extension/src/test/fixtures/expShow/noErrors.ts

```diff
@@ -0,0 +1,24 @@
+import { ExperimentFieldsOrError, ExperimentsOutput } from '../../../cli/reader'
+import expShowFixture, { errorShas } from './output'
+
+const excludeErrors = (): ExperimentsOutput => {
+  const { workspace, ...branchesObject } = expShowFixture
+  const expShowFixtureWithoutErrors: ExperimentsOutput = { workspace }
+
+  for (const [sha, { baseline, ...experimentsObject }] of Object.entries(
+    branchesObject
+  )) {
+    const experiments: { [sha: string]: ExperimentFieldsOrError } = {}
+    for (const [sha, experiment] of Object.entries(experimentsObject)) {
+      if (!errorShas.includes(sha)) {
+        experiments[sha] = experiment
+      }
+    }
+    expShowFixtureWithoutErrors[sha] = { baseline, ...experiments }
+  }
+  return expShowFixtureWithoutErrors
+}
+
+const data = excludeErrors()
+
+export default data
```

Comment:
[F] Without this extra fixture there would be a lot of changes to plots tests which are currently unrelated to this change.

Reply:
And this is why the likes of `extension/src/test/suite/plots/index.test.ts` are now pointed at this.

Reply:
When I come back to do plots errors I should be able to remove this. As we'll want to highlight the behaviour that we can have experiment errors but still try to plot those experiments.